src/ImageSharp/Processing/Processors/Quantization/OctreeQuantizer{TPixel}.cs

```diff
@@ -27,9 +26,9 @@ public struct OctreeQuantizer<TPixel> : IQuantizer<TPixel>
     private readonly int maxColors;
     private readonly int bitDepth;
     private readonly Octree octree;
-    private IMemoryOwner<TPixel> paletteOwner;
+    private IMemoryOwner<TPixel>? paletteOwner;
```

Comment:
Uninitialized instances of this struct are invalid.

```suggestion
    private IMemoryOwner<TPixel> paletteOwner;
```

Rendering `!` unnecessary in all `this.paletteOwner!` expressions below.

Reply:
Ok then I have to remove the setting of null in dispose